feral-lambda-runtime/src/main/scala/feral/lambda/runtime/LambdaReservedEnvVars.scala

```diff
@@ -0,0 +1,20 @@
+package feral.lambda.runtime
+
+import cats.effect.kernel.Sync
+
+object LambdaReservedEnvVars {
+  val HANDLER = "_HANDLER"
+  val AWS_REGION = "AWS_REGION"
+  val AWS_EXECUTION_ENV = "AWS_EXECUTION_ENV"
+  val AWS_LAMBDA_FUNCTION_NAME = "AWS_LAMBDA_FUNCTION_NAME"
+  val AWS_LAMBDA_FUNCTION_MEMORY_SIZE = "AWS_LAMBDA_FUNCTION_MEMORY_SIZE"
+  val AWS_LAMBDA_FUNCTION_VERSION = "AWS_LAMBDA_FUNCTION_VERSION"
+  val AWS_LAMBDA_LOG_GROUP_NAME = "AWS_LAMBDA_LOG_GROUP_NAME"
+  val AWS_LAMBDA_LOG_STREAM_NAME = "AWS_LAMBDA_LOG_STREAM_NAME"
+  val AWS_ACCESS_KEY_ID = "AWS_ACCESS_KEY_ID"
+  val AWS_SECRET_ACCESS_KEY = "AWS_SECRET_ACCESS_KEY"
+  val AWS_LAMBDA_RUNTIME_API = "AWS_LAMBDA_RUNTIME_API"
+  val LAMBDA_TASK_ROOT = "LAMBDA_TASK_ROOT"
+  val LAMBDA_RUNTIME_DIR = "LAMBDA_RUNTIME_DIR"
+  val TZ = "TZ"
+}
```

Comment:
Hi, I'm not sure I 100% understand this part. Isn't the `Env[F]` from `cats.effect.std` not included in the current version of cats effect? Or do you mean that I should create a custom `Env[F]` for the moment until it becomes available? 

Also, I'm not sure how I would implement the methods within the `LambdaRuntimeEnv[F]` trait. I would have assumed that the implementations of each of the methods would be given here
```scala
object LambdaRuntimeEnv {
	def apply[F[_]](env: Env[F]): LambdaRuntimeEnv[F] = new LambdaRuntimeEnv[F] {
		def lambdaRuntimeDir: F[String] = ???
		// ...
	}
}
```
instead of giving implementations directly within the trait.

Reply:
> Isn't the `Env[F]` from `cats.effect.std` not included in the current version of cats effect?

It is now available in v3.4.0-RC1, you can update to that and use it :)

> Also, I'm not sure how I would implement the methods within the `LambdaRuntimeEnv[F]` trait.

Sorry, I typed that out too hasty :) you are right, the implementations should go in the `apply` method as you wrote. Also I think I meant to write `apply[F[_]](implicit env: Env[F]) ...`